mix.exs

```diff
@@ -44,6 +45,8 @@ defmodule Phoenix.LiveDashboard.MixProject do
     [
       # Actual deps
       {:phoenix_live_view, "~> 0.16.0-dev", phoenix_live_view_opts()},
+      {:phoenix, "1.6.0-dev", github: "phoenixframework/phoenix", override: true},
+      {:phoenix_html, "3.0.0-dev", github: "phoenixframework/phoenix_html", override: true},
```

Comment:
Rebase and we can remove both those lines now. :)

Reply:
To be clear, Phoenix v1.5.10 was released with the support for function watchers.